### ISSUE TITLE
causative page has problen in smaller screen

### ISSUE THREAD

**Issue by 4WGH** [Clinical-Genomics/scout#3421]
![image](https://user-images.githubusercontent.com/12525201/173630709-4b47ee81-62c3-419f-bf57-3759154b1d33.png)

 I've  though that the page was not loading, till I've scrolled down and could see the table was below and not on the side.
If I move the same page on a larger screen the table more to the side.

Probably not a real issue but good to know?

Or should  a note notify te table is below?

**Comment by northwestwitch** [comment]
One could perhaps reduce the size of the left sidebar on smaller screens

**Comment by dnil** [comment]
Yes, either first collapsing it to an icon column, then to a drop down. One could consider making it a `nav`, in which case bootstrap would probably deal with the responsiveness. Compare the parial nav we have on the page; its really only the active item from the top menu, but it gets shuffled into a pretty decent drop down icon automagically by bootstrap:

<img width="656" alt="Screenshot 2022-06-15 at 07 57 06" src="https://user-images.githubusercontent.com/758570/173753359-2c255e3b-f5f0-48a6-b8e4-d635de8af474.png">

**Comment by dnil** [comment]
We could also go ahead and make the sidebar(s) Offcanvas. That would deal with this issue as well, though may take a little getting used to.

**Comment by dnil** [comment]
https://getbootstrap.com/docs/5.1/components/offcanvas/

**Comment by northwestwitch** [comment]
> https://getbootstrap.com/docs/5.1/components/offcanvas/

Cool

**Comment by dnil** [comment]
Come to think of it, all of the above is good, but its a pretty intense page. Why dont we just drop the sidebar? There is a top nav after all, and anyone scrolling on that page is unlikely to quickly want to go to another page from the sidebar.

**Comment by northwestwitch** [comment]
Yes, I was thinking the same!

**Comment by dnil** [comment]
We can then e.g. change all sidebars to Offcanvas when it feels motivated. I'll make a PR for the lazy fix. 😁
